qiskit/extensions/standard/h.py

```diff
@@ -78,9 +78,13 @@ def to_matrix(self):
 
 @deprecate_arguments({'q': 'qubit'})
 def h(self, qubit, *, q=None):  # pylint: disable=invalid-name,unused-argument
-    """Apply Hadamard (H) gate to a specified qubit (qubit).
-    An H gate implements a rotation of pi about the axis (x + z)/sqrt(2) on the Bloch sphere.
-    This gate is canonically used to rotate the qubit state from |0⟩ to |+⟩ or |1⟩ to |-⟩.
+    """Apply Hadamard (H) gate.
+
+    Applied to a specified qubit ``qubit``.
+    An H gate implements a rotation of :math:`\pi` about the axis
+    :math:`\frac{(x + z)}{sqrt(2)}` on the Bloch sphere. This gate is
+    canonically used to rotate the qubit state from :math:`\ket{0}` to
+    :math:`\ket{+}` or :math:`\ket{1}` to :math:`\ket{-}`.
```

Comment:
I know this is the right thing... but it's unfortunate that what renders nicely online in this case is much harder to parse if you're reading from the code. Oh well 🤷‍♀

Reply:
@SooluThomas figured out that this doesn't work either (it doesn't throw a warning, but it doesn't render correctly). Sphinx math mode doesn't support LaTeX's `\ket{}` so it's now `| \rangle`, I'm not sure what's harder to read :/

Reply:
I still see `\ket`?

Reply:
Oops, yep looks like: https://github.com/Qiskit/qiskit-terra/pull/3902/commits/38e5fefe59b63370e947fe501a696d9d2afd8850 missed a few spots. I'll go and clean them up now

Reply:
I also still see `\ket` used here (though the other gates use `\rangle`)